mon-pix/app/components/challenge-statement.js

```diff
@@ -21,7 +22,18 @@ export default class ChallengeStatement extends Component {
     if (!instruction) {
       return null;
     }
-    return instruction.replace('${EMAIL}', this._formattedEmailForInstruction());
+    const externalLinkRegex = /(\[(.*?)\]\((.*?)\))+/;
+    const formattedInstruction = instruction.replace('${EMAIL}', this._formattedEmailForInstruction());
+    return formattedInstruction.replace(externalLinkRegex, this._insertLinkTitle.bind(this));
```

Comment:
proposition : 
```suggestion
    const formattedEmailAndLinkInstruction = instruction.replace('${EMAIL}', this._formattedEmailForInstruction()).replace(externalLinkRegex, this._insertLinkTitle.bind(this));
    return formattedEmailAndLinkInstruction
```

ou 

```suggestion
    const formattedEmailInstruction = _formattedEmail(instruction);
    const formattedInstruction = _formattedLink(formattedEmailInstruction)
    return formattedInstruction;
```

Reply:
♥️ pour la deuxième suggestion.